### DISCUSSION TITLE
Possibile to customize t function?

### DISCUSSION THREAD

**Discussion by nephix** [i18next/i18next#1832] [Q&A] [answered]
Hi,

is it possible to customize the `t` function of `react-i18next`?

I'd like to have a "debug mode" in my app that simply renders the locale key a la 

```
t: (msg: string) => msg
```

I know there is a `debug` flag in `init()` but it's not what I'm looking for.

My use case is that I want to render all translation keys instead of the actual translation, so developers know what keys appear in certain locations if they want to edit them.

**Comment by jamuhl** [comment, marked as answer]
setting lng="cimode" does exactly that

**Reply by nephix** [reply]
Ah great thank you! I don't how I could have missed that in the docs 😅
